src/Microsoft.AspNetCore.Mvc.TagHelpers/EnvironmentTagHelper.cs

```diff
@@ -30,13 +34,33 @@ public EnvironmentTagHelper(IHostingEnvironment hostingEnvironment)
 
         /// <summary>
         /// A comma separated list of environment names in which the content should be rendered.
+        /// If the current environment is also in the <see cref="Exclude"/>  list, the content will not be rendered.
         /// </summary>
         /// <remarks>
         /// The specified environment names are compared case insensitively to the current value of
         /// <see cref="IHostingEnvironment.EnvironmentName"/>.
         /// </remarks>
         public string Names { get; set; }
```

Comment:
What if we made it a true alias for Include? Aka, have its getter/setter point to `Include`. That way we could add a doc comment stating it was an alias which would move people towards Include and avoid any confusion in the future.

It'd also simplify the code in process a decent amount.